### DISCUSSION TITLE
Is it possible to seek to a certain time of the video?

### DISCUSSION THREAD

**Discussion by scottmessinger** [muxinc/elements#418] [Q&A] [answered]
For instance, let's say I have two columns: one for the video and one with a list of chapters in the video. When someone clicks on a chapter, I want them taken to that point in the video.

Is there a way to control the player to advance it to a certain time?

**Comment by cjpillsbury** [comment, marked as answer]
Yes! Here's a very paired down example of how you might do that: https://codesandbox.io/s/mux-player-on-demand-simple-forked-ei8bzi

**Reply by scottmessinger** [reply]
Great!!! Thanks so much!
